source/index.js

```diff
@@ -220,8 +306,18 @@ class Storyblok {
     return this.client[type](url, params)
   }
 
-  newVersion() {
```

Comment:
@onefriendaday it seems to me that you haven't updated the type definitions in `types/index.d.ts`